### DISCUSSION TITLE
Flycast with an arcade cabinet setup

### DISCUSSION THREAD

**Discussion by MydknyteStyrm** [flyinghead/flycast#1158] [Q&A] [answered]
I am using four Ultrastik 360 USB joysticks and a IPAC4 for buttons. How can I incorporate both sets of controls for each player in the Config menu? On each joystick (under Arcade Controls) I can set the axis for the joystick, but the buttons (which act as a keyboard press) is not registering. I only can set buttons for the Keyboard option. Is there a way to incorporate everything together or do I need to use a third party app? Thanks!

**Comment by flyinghead** [comment, marked as answer]
You should assign your keyboard to port `All`. Then you can map each keyboard key to a given button and player using the port selection at the top of the window.

**Comment by MydknyteStyrm** [comment]
That did the trick. Thanks for the quick reply!

**Comment by CO-Windler** [comment]
Also coin slot 2 is not emulated. It looks like that Sega Driving Simulator (which contains no coin slots) has this input wired as an additional player control (STOP button?).
